src/Runner/FileLintingFileIterator.php

```diff
@@ -24,7 +24,7 @@
  *
  * @extends \IteratorIterator<mixed, \SplFileInfo, \Traversable<\SplFileInfo>>
  */
-final class FileLintingIterator extends \IteratorIterator
+final class FileLintingFileIterator extends \IteratorIterator implements LintingResultAwareFileIteratorInterface
```

Comment:
curious: why rename ?

Reply:
I believe it's related to the introduction of `LintingResultAwareFileIteratorInterface` and to have consistency and explicitness that it's a _file_ iterator.

Reply:
`File*Iterator` is already _file_ iterator 🤔

Reply:
IMHO grammar-wise, "file linting iterator" is an iterator that _lints files_, but not necessarily _iterarates over files_ 😅. But you're right about that double "file", maybe it should be `LintingFileIterator`? I will think about it.

Reply:
LintingFileIterator also OK